ci/run_clang_tidy.sh

```diff
@@ -1,12 +1,15 @@
 #!/bin/bash
-# SPDX-FileCopyrightText: Copyright (c) 2023-2024, NVIDIA CORPORATION.
+# SPDX-FileCopyrightText: Copyright (c) 2023-2025, NVIDIA CORPORATION.
```

Comment:
Given this file was updated, the style check wanted a newer year in the header

Went ahead and made this change. Hope that is ok